src/index.test.js

```diff
@@ -18,6 +19,9 @@ test('Can instantiate the plugin with no options', () => {
   expect(_.isPlainObject(inst.config)).toBe(true);
   expect(inst.timeline instanceof Perf).toBe(true);
   expect(inst.config.autoMeasure).toBe(true);
+  expect(inst.meta.name).toBe(pkg.name);
```

Comment:
@kolanos would it be cool to change pkg.name to a string value for a bit of extra validation?